nixos/modules/services/web-apps/convos.nix

```diff
@@ -0,0 +1,80 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.convos;
+in
+{
+  options.services.convos = {
+    enable = mkEnableOption "Convos";
+    package = mkOption {
```

Comment:
It doesn't seem like there would ever be a practical case to override the default here, so I would recommend dropping it.